Documentation/CRDs/Block-Storage/ceph-block-pool-rados-namespace-crd.md

```diff
@@ -81,3 +89,43 @@ parameters:
   pool: replicapool
   ...
 ```
+
+### Mirroring
+
+First, enable the mirroring in the default rados namespace of the CephBlockPool that contains the new rados namespace.
```

Comment:
What does "default rados namespace" mean? Will users know what this means?

IMO, it would be clearer to say "First, enable mirroring for the parent CephBlockPool". I understand that there is some nuance here, but 90% of users won't understand the nuance, and users who do probably/hopefully won't be tripped up.